### ISSUE TITLE
Docker container?

### ISSUE THREAD

**Issue by longnguyen2004** [MaxOhn/Bathbot#3]
The installation process could be massively simplified and automated with a Docker container. Are there any plans for it?

**Comment by MaxOhn** [comment]
Haven't considered that yet, will look into it

**Comment by MaxOhn** [comment]
I finally found the motivation to get the hang of docker and added a compose file to set up the two external dependencies postgres an redis.
That way neither of the two are required to be installed, be it for development of bathbot or just running the bot.

However, I decided against making and publish an image for the bot itself since it's useless for my own workflow and although I'm fine with other people running the bot, I also don't feel like putting too much effort into helping people run the bot.
Maybe I'll reconsider it sometime.

So basically the only remaining pain point to run the bot yourself is the rust compilation.
